tasks/install-service

```diff
@@ -62,7 +62,7 @@ ExecStart=/usr/bin/podman run --name=cockpit-tasks-%i --hostname=${CONTAINER_HOS
     --device=/dev/kvm --network=cockpit-tasks-%i \
     --memory=24g --pids-limit=16384 --shm-size=1024m ${TMPVOL:-} \
     --volume=\${TEST_CACHE}/images:/cache/images:rw \
-    --volume=\${TEST_SECRETS}/tasks:/secrets:ro \
+    --volume=\${TEST_SECRETS}/tasks:/run/secrets/tasks:ro \
```

Comment:
I wonder if we could think a step ahead here and:

 - add `/run/secrets/github-token` as the 'authoritative' location for the token file
 - add `/run/secrets/s3-keys` as the location for the s3-keys directory

...and adjust the symlinks to point to those things.  That's where we'll want to have those two things inside of containers spawned by `job-runner`, under control of the "secrets" attribute in the job definition.

Of course, we'd also continue to have the volume mount for `secrets/webhook` and `secrets/tasks` since we still use those at present.  We might want to start thinking about what to do more generally with "external config" such as the npm-registry certificate, though... it's not really a secret.

Reply:
No objection to the two extra volumes (and doing the same in `run-local.sh`). Do you want to do that in this PR?

I'm not sure if we can commit the npm registry internal RH mirror cert to a public place -- it's not really secret, but it applies to RH internal infrastructure only. TBH I wouldn't change that right now without at least some more checking.

Reply:
I'd like to see the github-token and s3-keys bit in this PR.  No time like the present!

The npm registry cert is a bit more interesting and it dovetails into the question about what to do with the npm config bits that current live in `setup-tasks` (which needs to die, I'm increasingly convinced)...

Reply:
I didn't yet do this, instead used the env variables. Let's stabilize this version first, then I can take another look.

Reply:
Sure.  With the envvars, I care a lot less.